packages/jest-cli/src/TestRunner.js

```diff
@@ -364,10 +364,14 @@ class TestRunner {
 
   _setupReporters() {
     const config = this._config;
-
-    this.addReporter(
-      config.verbose ? new VerboseReporter(config) : new DefaultReporter(),
-    );
+    
+    
+    // Default Reporters are setup when
+    // noDefaultReporters is false, which is false by default
+    // and can be set to true in configuration.
```

Comment:
I think this comment is superfluous as it describes what the if statement does. Mind removing it?